CHANGELOG.md

```diff
@@ -5,10 +5,10 @@ All notable changes to this project will be documented in this file, per [the Ke
 ## [Unreleased] - TBD
 
 ## [1.2.1] - 2024-08-22
-**Note that this release bumps the WordPress minimum version from 5.8 to 6.4.**
+**Note that this release bumps the WordPress minimum version from 5.8 to 6.6.**
```

Comment:
So I don't think this is accurate, assuming I'm reading this correctly. We bumped the tested version to 6.6 but the minimum version is 6.4, which is what we call out here. So I think this should be left as-is